src/matcha_ml/core/core.py

```diff
@@ -8,6 +9,31 @@
 from matcha_ml.state import MatchaStateService, RemoteStateManager
 
 
+def get_local_state_hash(matcha_state_path: str) -> str:
```

Comment:
See below RE naming.

also, would this be more consistent as a method within the `MatchaStateService`, as we're using an instance of this in `core.py` already?

Reply:
I would agree with this

Reply:
Moved `get_hash_local_state` function to `MatchaStateService`.